modules/local/samplesheet_check.nf

```diff
@@ -22,7 +22,11 @@ process SAMPLESHEET_CHECK {
     check_samplesheet.py \\
         $samplesheet \\
         samplesheet.valid.csv
-
+    if("${params.aligner}"=="cellranger") {
+        samplesheet_cellranger_conform.py \\
+        samplesheet.valid.csv \\
+        samplesheet_cellranger.csv
+    }
```

Comment:
The `if` here would need to be written in `bash`. 

Alternatively, you put the `if` outside the `"""` block and assign the result in a variable that can be included in the scripts block.